certbot/CHANGELOG.md

```diff
@@ -11,8 +11,14 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 ### Changed
 
 * `acme.client.ClientBase`, `acme.messages.Authorization.resolved_combinations`,
-  `acme.messages.Authorization.combinations` and `acme.mixins` are deprecated and
-  will be removed in a future release.
+  `acme.messages.Authorization.combinations`, `acme.mixins`, `acme.fields.resource`,
+  and `acme.fields.Resource` are deprecated and will be removed in a future release.
+* `acme.messages.OLD_ERROR_PREFIX` (`urn:acme:error:`) is deprecated and support for
+  the old ACME error prefix in Certbot will be removed in the next major release of
+  Certbot.
+* `acme.messages.Directory.register` is deprecated and will be removed in the next
+  major release of Certbot. Furthermore, `.Directory` will only support lookups
+  by the exact resource name string in the ACME directory  (e.g. `directory['newOrder']`).
```

Comment:
Should we add a warning about this to `acme.messages.Directory.__getitem__`? I think we could do this by printing a warning if the `name` parameter there is not a string.